### PR TITLE
Modified the number 1 to the number 2 inside test_file.txt

### PR DESCRIPTION
**Pull request by Dima12A**
this is my first pull request special for Java_Fundamentals ;)

### CODE REVIEW

**Review by Dima12A — COMMENTED** [review]
typo in file name, correct file name - first_file.txt

File: first_file.txt [hunk 1]
```diff
@@ -1 +1 @@
-1
\ No newline at end of file
+3
```
Comment (gh3rtzz) [inline first_file.txt]: Nice. Now change it to 4

Comment (Dima12A) [reply first_file.txt]: done

**Review by gh3rtzz — APPROVED** [review]
Approved. Go ahead and merge it

**Pull request merged**